<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CutsceneWidget</class>
 <widget class="QWidget" name="CutsceneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <property name="spacing" stdset="0">
      <number>2</number>
     </property>
     <property name="leftMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="topMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="rightMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="bottomMargin" stdset="0">
      <number>2</number>
     </property>
     <widget class="QSplitter" name="splitterLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidgetBottom">
       <layout class="QVBoxLayout" name="verticalLayoutBottom">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QToolBar" name="toolBarEntities">
          <property name="floatable">
           <bool>false</bool>
          </property>
          <addaction name="actionAddSegment"/>
          <addaction name="actionRemoveSegment"/>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="container" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QtTreePropertyBrowser" name="propertyBrowser" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RichTextEditor" name="richTextEditor" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="NovelTeaWidget" name="preview" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddSegment">
   <property name="icon">
    <iconset theme="add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Segment</string>
   </property>
  </action>
  <action name="actionRemoveSegment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove Segment</string>
   </property>
  </action>
  <action name="actionAddText">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionAddPageBreak">
   <property name="text">
    <string>Page Break</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NovelTeaWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/NovelTeaWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>Widgets/QtPropertyBrowser/qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RichTextEditor</class>
   <extends>QWidget</extends>
   <header>Widgets/RichTextEditor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
